test/servico/cst09Test.java

```diff
@@ -71,7 +71,7 @@ public void tearDown() {
     public void testeEfetuarRealocação_SemSubstitutosDisponíveis() throws ParseException
     {
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Administrador", "123456"));
+        assertTrue(loginService.verificarUsuarioESenha("Administrador", "123456"));

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 83).](https://www.codacy.com/app/calebepb/substituicao/file/2738501517/issues/source?bid=3308597&fileBranchId=3353492#l74)